platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/connectivity/SocketFactory.java

```diff
@@ -126,4 +128,13 @@ static void configureAndConnect(
      */
     @NonNull
     Socket createClientSocket(@NonNull final String hostname, final int port) throws IOException;
+
+    /**
+     * A convenient handler for executing logic on dynamic peers
+     * e.g. it could be used to re-initialize certificate keystore, or ask peers to re-authenticate
+     *
+     * @param peerInfoList
+     *  the updated list of peers
```

Comment:
Javadoc line wrapping is a bit funky. Suggest using intellij to auto-format.

Reply:
thanks. I've turned this on so it's automatic on commit.